main.py

```diff
@@ -0,0 +1,19 @@
+import xml.etree.ElementTree as ET
+root = ET.parse('feed.xml').getroot()
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
```

Comment:
No need to keep unused `bcolors` class properties.

```suggestion
class bcolors:
    WARNING = '\033[93m'
    ENDC = '\033[0m'
```

Reply:
Done